### PR TITLE
Remove obsolete lines of text/code

### PR DESCRIPTION
**Pull request by binaryflesh**
see #236

### CODE REVIEW

**Comment by NashMiao** [comment]
I'll take time to review these code, please give me some time, thanks for your contribution to `ontology-python-sdk`.

File: ontology/account/account.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def __init__(self, private_key: str or bytes, scheme=SignatureScheme.SHA256withE
 
     def generate_signature(self, msg: bytes) -> object:
         """
-        Generate and return verified message signed with key pair.
+        Generate verified message signed with key pair.
```
Comment (NashMiao) [inline ontology/account/account.py]: Generate message signature in account's signature scheme.

Comment (NashMiao) [inline ontology/account/account.py]: Generate message signature in account's signature scheme.

Comment (binaryflesh) [reply ontology/account/account.py]: Happy accident. fixed.

File: ontology/account/account.py [hunk 2]
```diff
@@ -20,75 +21,59 @@
 from ontology.crypto.signature_scheme import SignatureScheme
 from ontology.crypto.signature_handler import SignatureHandler
 
+_SHA256 = SignatureScheme.SHA256withECDSA
+_SHA3_224 = SignatureScheme.SHA3_224withECDSA
+_SHA3_384 = SignatureScheme.SHA3_384withECDSA
+_SHA512 = SignatureScheme.SHA512withECDSA
+
+SUPPORTED_SCHEMES = [_SHA256, _SHA3_224, _SHA3_384, _SHA512]
```
Comment (NashMiao) [inline ontology/account/account.py]: Could you explain to me why this change is necessary? In fact, more signature scheme will be supported in the future. I think this simplification will lead to confusion in the future.

**Pull request closed without merging**